wkd/kustomization.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: wkd
+resources:
+  - github.com/drGrove/docker-wkd/kustomize/advanced?ref=9e0721ef88ed33aac78533bbb5e21f7d9a0c69cd
+  - ingress.yaml
+patches:
+  - target:
+      kind: Deployment
+      name: wkd
+    patch: |-
+      - op: add
+        path: "/spec/template/spec/containers/0/args/-"
+        value: "-m"
+      - op: add
+        path: "/spec/template/spec/containers/0/args/-"
+        value: "hashbang.sh"
+images:
+  - name: drgrove/wkd
+    digest: sha256:b00e9e42486efa6e8601f4aff3d61c9417dcaba993411905a4b63cd2dee72f42 # v2.2.2
+  - name: kiwigrid/k8s-sidecar
+    digest: sha256:9b6e173c61ca4a9470a52729f362dabd5962581d85d9e6746dcfb9fcfa484b42 # 1.21.0
```

```suggestion
  - name: kiwigrid/k8s-sidecar:1.21.0
    digest: sha256:9b6e173c61ca4a9470a52729f362dabd5962581d85d9e6746dcfb9fcfa484b42
```

by doing it this way we can make sure we don't accidentally downgrade on the next bump

---

I don't think this will work because the upstream has `:latest`